### ISSUE TITLE
1.5.0-beta14. Android: replace_value sets a corrupted value

### ISSUE THREAD

**Issue by TikhomirovSergey** [appium/appium#6142]
Client code sample:

```java
    AppiumDriverLocalService service = AppiumDriverLocalService.buildDefaultService();
    service.start();

    File appDir = new File("src/test/java/io/appium/java_client");
    File app = new File(appDir, "ApiDemos-debug.apk");
    DesiredCapabilities capabilities = new DesiredCapabilities();
    capabilities.setCapability(MobileCapabilityType.BROWSER_NAME, "");
    capabilities.setCapability(MobileCapabilityType.DEVICE_NAME, "Android Emulator");
    capabilities.setCapability(MobileCapabilityType.PLATFORM_NAME, MobilePlatform.ANDROID);
    capabilities.setCapability(MobileCapabilityType.APP, app.getAbsolutePath());
    AndroidDriver<AndroidElement> = new AndroidDriver<AndroidElement>(service.getUrl(), capabilities);

...
    AndroidElement editElement = driver.findElementByAndroidUIAutomator("resourceId(\"io.appium.android.apis:id/edit\")");

    editElement.sendKeys(originalValue);

    assertEquals(originalValue, editElement.getText());

    editElement.replaceValue(replacedValue);
```

There is no error but the result is

![corrupted_replace_value](https://cloud.githubusercontent.com/assets/4927589/13047461/4dc9f088-d3f0-11e5-8cd0-2f88d6ef9d65.png)

Server output:
https://gist.github.com/TikhomirovSergey/b7c824034e27f3a63284

**Comment by TikhomirovSergey** [comment]
appium/java-client#302
appium/appium-dotnet-driver#136

**Comment by imurchie** [comment]
Fixed in `appium-android-driver` 1.6.9.
